runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientFetchFactory.java

```diff
@@ -290,7 +292,8 @@ public MessageConsumer newStream(
                     leaderId,
                     initialOffset,
                     isolation,
-                    sasl)::onApplication;
+                    sasl,
+                    catalog)::onApplication;
```

Comment:
Unlikely that we would put the config so deep into the stream, instead it would probably be converted to behavior first, such as `Validator`.